_posts/2024-07-31-openscanhub-fedora-and-packit.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OpenScanHub, Fedora and Packit
+layout: post
+category: posts
+---
+
+TLDR: We have deployed OpenScanHub on the [Fedora infrastructure](https://openscanhub.fedoraproject.org/) and recently integrated with [Packit](https://packit.dev/). An example scan can be seen on [GitHub](https://github.com/strace/strace/pull/300/checks?check_run_id=28180162251) and more information can be found in the [OpenScanHub](https://fedoraproject.org/wiki/OpenScanHub) and [Packit](https://packit.dev/docs/configuration) documentation.
```

Comment:
```suggestion
TLDR: We have deployed OpenScanHub on the [Fedora infrastructure](https://openscanhub.fedoraproject.org/) and recently integrated with [Packit](https://packit.dev/). An example scan can be seen on [GitHub](https://github.com/strace/strace/pull/300/checks?check_run_id=28180162251) and more information can be found in the [OpenScanHub](https://fedoraproject.org/wiki/OpenScanHub) and [Packit](https://packit.dev/docs/configuration#osh_diff_scan_after_copr_build) documentation.
```

Reply:
Is there a chance this field name may change and break the link later? I wanted to avoid breaking links in the blog post.